CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+
+# Contributing to `{project_name}`
+
+Thank you for your interest in becoming a part of the ongoing development of `{project_name}`.
+
+This guide covers our practices and recommendations for contributing to the project.
+
+
+## Table of Contents
+
+[Ways to contribute](#ways-to-contribute)
+
+[Making your first contribution](#making-your-first-contribution)
+
+[Pull requests](#pull-requests)
+
+[Commit messages](#commit-messages)
+
+[Code of Conduct](#code-of-conduct)
+
+
+
+## Ways to contribute
+
+There are various ways to get involved, here are a few suggestions of things we'd love some help with.
+
+#### Resolving existing issues
+
+ You can consider helping out with [issues](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) already requiring attention.
```

Comment:
if "project_name" is dynamic, can the URL of the project be dynamic also?

if not, maybe:
```suggestion
 You can consider helping out with [issues](./issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) already requiring attention.
```
?